app/builders/messages/facebook/message_builder.rb

```diff
@@ -120,7 +121,7 @@ def message_params
       message_type: @message_type,
       content: response.content,
       source_id: response.identifier,
-      sender: contact
+      sender: @outgoing_echo ? nil : contact
```

Comment:
should we need the instance variable ? or can we just use the `outgoing_echo` function?